internal/changelog/entry.go

```diff
@@ -12,16 +12,14 @@ type FragmentFileInfo struct {
 }
 
 type Entry struct {
-	// fields from template
-	Kind        Kind   `yaml:"kind"`
-	Summary     string `yaml:"summary"`
-	Description string `yaml:"description"`
-	Component   string `yaml:"component" `
-	LinkedPR    int    `yaml:"pr"`
-	LinkedIssue int    `yaml:"issue"`
-
-	Timestamp int64            `yaml:"timestamp"`
-	File      FragmentFileInfo `yaml:"file"`
+	Summary     string           `yaml:"summary"`
+	Description string           `yaml:"description"`
+	Component   string           `yaml:"component" `
+	Kind        Kind             `yaml:"kind"`
+	LinkedPR    []int            `yaml:"pr"`
+	LinkedIssue int              `yaml:"issue"`
+	Timestamp   int64            `yaml:"timestamp"`
+	File        FragmentFileInfo `yaml:"file"`
```

Comment:
I'm making this suggestion to avoid changing the sort order of fields in this struct. It makes more difficult to check what changed in the block and the previous sort order was based on sort order of fields in the template (making it easy to cross check and find values).

Could you please update this to keep the previous order?

```suggestion
	// fields from template
	Kind        Kind             `yaml:"kind"`
	Summary     string           `yaml:"summary"`
	Description string           `yaml:"description"`
	Component   string           `yaml:"component" `
	LinkedPR    []int            `yaml:"pr"`
	LinkedIssue int              `yaml:"issue"`

	Timestamp int64            `yaml:"timestamp"`
	File      FragmentFileInfo `yaml:"file"`
```

Reply:
I agree, I've left it as per the template.